fri:
  port: 30200
  robot_ip: 192.170.10.2
  robot_description: /robot_description

hardware_interface:
  control_freq: 1000 # in Hz
  joints:
    - iiwa_joint_1
    - iiwa_joint_2
    - iiwa_joint_3
    - iiwa_joint_4
    - iiwa_joint_5
    - iiwa_joint_6
    - iiwa_joint_7
